packages/sdk/src/swap/v2/types.ts

```diff
@@ -0,0 +1,183 @@
+import {
+  AffiliateBroker,
+  FillOrKillParams,
+  SwapFee,
+  type CcmParams,
+  type DcaParams as Dca,
+} from '@/shared/schemas';
+import { SwapType } from '@/swap/client';
+import { FailureMode } from '@/swap/utils/swap';
+import { ChainsAndAssets } from '../types';
+
+// eslint-disable-next-line @typescript-eslint/no-namespace
+export namespace SwapV2 {
+  type DcaParameters = Omit<Dca, 'chunkInterval'> & {
```

Comment:
think i would be explicit here and add all of the properties explicitly instead of combining types